wit/blob-types.wit

```diff
@@ -0,0 +1,35 @@
+// name of a container, a collection of objects.
+// The container name may be any valid UTF-8 string.
+type container-name = string
+
+// name of an object within a container
+// The object name may be any valid UTF-8 string.
+type object-name = string
+
+type timestamp = u64
+
+// information about a container
+record container-metadata {
+	// the container's name
+	name: container-name,
+	// date and time container was created
+	created-at: timestamp,
+}
+
+// information about an object
+record object-metadata {
+	// the object's name
+	name: object-name,
+	// the object's parent container
+	container: container-name,
+	// date and time the object was created
```

Comment:
u64 is a date time? Is this a resource pointer or is the date time? I see the usage with a TODO comment. Can you expand on this?

Reply:
It is date & time, and I implemented it by converting the date & time struct from service response to secs.

Reply:
Ok, but secs from what base? Since epoch, big bang, or some other notable event?

Reply:
Just to be clear, the comment above was meant half as comedy and the other half as a serious question.

Reply:
Oh, since UNIX Epoch

Reply:
Please consider calling that out in the description of that field / parameter.

Reply:
I guess you are suggesting for a change in the wasi-blob-store spec where we can define a `date_and_time` resource which provides functions to explictly convert date and time to some other representations.

Like secs from unix epoch as a `i64`, or nano seconds as a `i32`

Reply:
I didn't see the `date_and_time` resource, but any place I see a field for datetime as an numeric it (imho) deserves to have some explanation of how that is converted from a calendar datetime to an integer to be clear about the expectations of the API. Perhaps, pointing to functionality to transform from / to datetime from / to numeric.

Reply:
Okay raised this issue to track https://github.com/WebAssembly/wasi-blob-store/issues/7